Cargo.toml

```diff
@@ -79,10 +79,6 @@ tracing-subscriber = { version = "0.2.20", features = ["env-filter"] }
 [target.'cfg(unix)'.dependencies]
 skim = "0.9.4"
 
-[build-dependencies]
-clap = "3.0.0-rc.7"
-man = "0.3.0"
```

Comment:
btw we have an active PR for creating a `clap_man` crate

https://github.com/clap-rs/clap/pull/3174